x-pack/legacy/plugins/siem/public/components/timeline/body/column_headers/helpers.test.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getActionsColumnWidth, getColumnWidthFromType } from './helpers';
+import {
+  DEFAULT_COLUMN_MIN_WIDTH,
+  DEFAULT_DATE_COLUMN_MIN_WIDTH,
+  DEFAULT_ACTIONS_COLUMN_WIDTH,
+  SHOW_CHECK_BOXES_COLUMN_WIDTH,
+  EVENTS_VIEWER_ACTIONS_COLUMN_WIDTH,
+} from '../constants';
+
+describe('helpers', () => {
```

Comment:
🙏 thank you for adding tests!

Reply:
OK, just realized that you didn't _add_ them per se, but: 🙏 thank you for not deleting them!